src/TabSwipe/widget/TabSwipe.webmodeler.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, createElement } from "react";
+
+// tslint:disable-next-line class-name
+export class preview extends Component<{}, {}> {
+
+    render() {
+        return createElement("div", { className: "widget-tab-swipe-preview" });
```

Comment:
empty preview should have a class make it clear that it is intentional empty. use class name `widget-tab-swipe-hide-preview"`